### ISSUE TITLE
Training model on Safari IOS return NaN

### ISSUE THREAD

**Issue by PSSABISHEK** [tensorflow/tfjs#3267]
#### TensorFlow.js version
1.7.2

#### Browser version
Safari IOS - 13

#### Describe the problem or feature request
Backend Used - WebGL
Loss is all NaN values while training a model (MobileNetv2) and accuracy is very poor (0) on IOS Safari. The same model trains and performs extremely well on android devices and on desktop browsers (including Safari)

Epochs Run - 10

![image](https://user-images.githubusercontent.com/13751214/81839078-af632500-9564-11ea-9bc2-d94b00cf0e16.png)

**Comment by rthadur** [comment]
Hello, this sort of question is better suited for StackOverflow. This issues list is for bugs and feature requests. Thanks and please use the tag 'tensorflow.js' on your StackOverflow question to makes sure it's discoverable.
